barretenberg/cpp/src/barretenberg/dsl/acir_format/serde/acir.hpp

```diff
@@ -239,8 +239,6 @@ struct BlockId {
     static BlockId bincodeDeserialize(std::vector<uint8_t>);
 };
 
-// TODO(https://github.com/AztecProtocol/barretenberg/issues/825): This struct is more general than it needs / should be
-// allowed to be. We can only accommodate 1 quadratic term and 3 linear terms.
```

Comment:
I think we want to keep this?

Reply:
This TODO probably can't live here, it should exist I guess somewhere in the handling of this struct? not sure I fully get the issue